liquidctl/cli.py

```diff
@@ -11,39 +11,43 @@
-  --serial <number>           Filter devices by serial number
-  --bus <bus>                 Filter devices by bus
-  --address <address>         Filter devices by address in bus
-  --usb-port <port>           Filter devices by USB port in bus
-  -d, --device <id>           Select device by listing id
+  -m, --match <substring>        Filter devices by description substring
+  -n, --pick <number>            Pick among many results for a given filter
+  --vendor <id>                  Filter devices by hexadecimal vendor ID
+  --product <id>                 Filter devices by hexadecimal product ID
+  --release <number>             Filter devices by hexadecimal release number
+  --serial <number>              Filter devices by serial number
+  --bus <bus>                    Filter devices by bus
+  --address <address>            Filter devices by address in bus
+  --usb-port <port>              Filter devices by USB port in bus
+  -d, --device <id>              Select device by listing id
 
 Animation options (devices/modes can support zero or more):
-  --speed <value>             Abstract animation speed (device/mode specific)
-  --time-per-color <value>    Time to wait on each color (seconds)
-  --time-off <value>          Time to wait with the LED turned off (seconds)
-  --alert-threshold <number>  Threshold temperature for a visual alert (°C)
-  --alert-color <color>       Color used by the visual high temperature alert
+  --speed <value>                Abstract animation speed (device/mode specific)
+  --time-per-color <value>       Time to wait on each color (seconds)
+  --time-off <value>             Time to wait with the LED turned off (seconds)
+  --alert-threshold <number>     Threshold temperature for a visual alert (°C)
+  --alert-color <color>          Color used by the visual high temperature alert
+  --direction <string>           If the pattern should move forward or backwards. [default: forward]
+  --start-led <number>           The first led to start the effect at
+  --maximum-leds <number>        The number of LED's the effect should apply to
 
 Other device options:
-  --single-12v-ocp            Enable single rail +12V OCP
-  --pump-mode <mode>          Set the pump mode (certain Corsair coolers)
-  --legacy-690lc              Use Asetek 690LC in legacy mode (old Krakens)
-  --non-volatile              Store on non-volatile controller memory
-  --unsafe <features>         Comma-separated bleeding-edge features to enable
+  --single-12v-ocp               Enable single rail +12V OCP
+  --pump-mode <mode>             Set the pump mode (certain Corsair coolers)
+  --legacy-690lc                 Use Asetek 690LC in legacy mode (old Krakens)
+  --non-volatile                 Store on non-volatile controller memory
+  --unsafe <features>            Comma-separated bleeding-edge features to enable
+  --temperature-sensor <number>  The temperature sensor number for the Commander Pro
```

Comment:
Use of `--temperature-sensor 2` seems to work correctly (no output), but it's falling back to first sensor. Isn't there any mismatch of parameter names? Because line 107 in this file says `--temp-sensor`, but while trying to use `--temp-sensor` I'm getting help output (seems like some kind of validation when parameter name isn't correct).

Tested on c94d931 build created by CI

Reply:
```
PS C:\liquidctl\dev\commander> ./liquidctl --match commander set fan1 speed 0 1 30 1500 --temp-sensor 2
Usage:
  liquidctl [options] list
  liquidctl [options] initialize [all]
  liquidctl [options] status
  liquidctl [options] set <channel> speed (<temperature> <percentage>) ...
  liquidctl [options] set <channel> speed <percentage>
  liquidctl [options] set <channel> color <mode> [<color>] ...
  liquidctl --help
  liquidctl --version

Reply:
Good catch. I missed that change.

Reply:
I think the latest commit has a fix for that.

Reply:
Works as expected, thanks for extremely fast fix :) I decided to buy Commander Pro when I've noticed that liquidctl integration is in progress, great job! :)

Reply:
good, Im happy it worked as anticipated for you.